index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Startup Matcher</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400i,700,700i,900" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Oswald:300,400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900" rel="stylesheet">
+  </head>
+  
+  <body>
+    <div class="container">
+
+      
+      <header>
+        <div class="logo">Startup Matchmaker</div>
+
+        <nav>
+          <a href="#" class="developers">Find Developers</a>
+          <a href="#" class="designers">Find Designers
+          </a>
+          <a href="#">How it Works</a>
+          <a href="#">Our Team</a>
+          <a href="#">Blog</a>
+        </nav>
+
+      </header>
+
+       
+
+      <div class="quote">
+        <h1>Because two brains <span class="taste">taste</span> <span class="are">are</span> better than one.</h1>
+      </div> 
+
+      <article>
+        <div class="image">
+          <img src="images/coworking.jpg" width="1024" height="664">
```

Comment:
its generally easier to manipulate images taken the space of a container with css. 
```
background: url( path/to/image );
background-size: cover;
```